examples/arrays/arrays.description

```diff
@@ -1,6 +1,3 @@
 // Arrays in Ballerina are mutable lists of values of dynamic length where each member of the list is
 // specified with the same type. The implicit initial value of an array is the array with a length of `0`.
-//
-// A Sealed Array in Ballerina is a list of values of fixed length, where each member value of the list is
-// specified with the same type.
-// The size of the sealed array should be specified at the point of declaration and can not be changed afterwards.
+// The length of an array can be fixed by providing the length at the point of declaration.
```

Comment:
Shall we change as follows:
"// In Ballerina, arrays are mutable lists of values with dynamic length, where each member in the list is
// of the same type. The implicit initial value of an array is the array with a length of `0`.	// If you want to have a fixed length array, specify the length of the array at the time of declaration."